src/test_worldgen.py

```diff
@@ -15,6 +15,9 @@
     print(f'Ruled by: {scenario.ruler.title} {scenario.ruler.name}')
     print(scenario.kingdom.description, '\n')
     print(scenario.ruler.governance_style, scenario.ruler.evil_deeds)
+    for region in scenario.kingdom.regions:
```

Comment:
This is fine for test code, but the world generation we're demoing probably shouldn't include this, to leave some surprises to players :)

Reply:
I thought this is just for test, not part of the demo